DigitalOffice/Services/UserService/Startup.cs

```diff
@@ -42,9 +52,39 @@ public void ConfigureServices(IServiceCollection services)
             ConfigureMappers(services);
         }
 
+        private void ConfigRabbitMq(IServiceCollection services)
+        {
+            string serviceId = "123";
+            string serviceName = "UserService";
+
+            services.AddMassTransit(x =>
+            {
+                x.AddConsumer<UserConsumer>();
+
+                x.UsingRabbitMq((context, cfg) =>
+                {
+                    cfg.Host("localhost", host =>
+                    {
+                        host.Username($"{serviceName}");
+                        host.Password($"{serviceName}_{serviceId}");
+                    });
+
+                    cfg.ReceiveEndpoint($"{serviceName}_AuthenticationService", ep =>
```

Comment:
Could u explain why call it that?

Reply:
What exactly, line?

Reply:
Most likely so, I also paid attention to this, but decided not to write.

Reply:
why are you call the url for consumers?